docs/references/config/config.toml.md

```diff
@@ -0,0 +1,2055 @@
+mempool for seen transactions. The mempool cache provides a list of transactions already received to filter out
+incoming duplicate transactions and prevent duplicate full transaction validations.
+
+### mempool.type
+The type of mempool this node will use.
+```toml
+type = "flood"
+```
+
+| Value type          | string    |
+|:--------------------|:----------|
+| **Possible values** | `"flood"` |
+|                     | `"nop"`   |
+
+`"flood"` is the original mempool implemented for CometBFT. It is a concurrent linked list with flooding gossip
+protocol.
+
+`"nop"` is a "no operation" or disabled mempool, where the ABCI application is responsible for storing, disseminating and
+proposing transactions. Note, that it requires empty blocks to be created:
+[`consensus.create_empty_blocks = true`](#consensuscreate_empty_blocks) has to be set.
+
+### mempool.recheck
+Transaction validity check.
+```toml
+recheck = true
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `true`  |
+|                     | `false` |
+
+Committing a block affects the application state, hence the remaining transactions in the mempool after a block commit
+might become invalid. Setting `recheck = true` will go through the remaining transactions and remove invalid ones.
+
+### mempool.broadcast
+Broadcast the mempool content (uncommitted transactions) to other nodes.
+```toml
+broadcast = true
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `true`  |
+|                     | `false` |
+
+This ensures that uncommitted transactions have a chance to reach multiple validators and get committed by one of them.
+
+Setting this to `false` will stop the mempool from relaying transactions to other peers until they are included in a
+block. Only the peer you send the tx to will see it until it is included in a block.
+
+### mempool.wal_dir
+Mempool write-ahead log folder path.
+```toml
+wal_dir = ""
+```
+
+| Value type          | string                                          |
+|:--------------------|:------------------------------------------------|
+| **Possible values** | relative directory path, appended to `$CMTHOME` |
+|                     | absolute directory path                         |
+|                     | `""`                                            |
+
+In case `$CMTHOME` is unset, it defaults to `$HOME/.cometbft`.
+
+Configures the location of the Write Ahead Log (WAL) for the mempool. `""`  disables the WAL.
+
+### mempool.size
+Maximum number of transactions in the mempool.
+```toml
+size = 5000
+```
+
+| Value type          | integer |
+|:--------------------|:--------|
+| **Possible values** | &gt;= 0 |
+
+If the mempool is full, incoming transactions are dropped.
+
+The value `0` is undefined.
+
+### mempool.max_txs_bytes
+The maximum size of all transactions accepted in the mempool.
+```toml
+max_txs_bytes = 1073741824
```

Comment:
Hadn't we decided to reduce this to N times the default block size (N being around 10?)

Reply:
Let's open an issue on this, and dealt with this text there

Reply:
https://github.com/cometbft/cometbft/issues/2756